ui-lib.pl

```diff
@@ -1442,7 +1442,7 @@ =head2 ui_buttons_start
 sub ui_buttons_start
 {
 return &theme_ui_buttons_start(@_) if (defined(&theme_ui_buttons_start));
-return "<table width='100%' class='ui_buttons_table'>\n";
+return "<table width='100%' class='ui_buttons_table'>\n<tr><td>";
```

Comment:
I believe so.

ui_buttons_row needs to also have a full table/tr/td wrapping the form. It is invalid to have a table/form/tr/td, and it breaks event-handling on the page when loaded via JavaScript (this is the bug Ilia was seeing in Authentic, and we'd see if here, too if we were to load the page via XHR or fetch or whatever JS loader).

I think I'd like to get rid of ui_buttons start. It's just an extraneous table wrapping the whole thing...it's a unneeded extra layer of table nesting. But, tons of stuff calls it, and I suspect making it a no-op would cause formatting problems. But, it's probably worth testing.

So, in short: A table can be contained inside of a tr/td, but a form cannot be contained in a table (without a tr/td outside of it). As it stands, we either need this to be a fully formed table wrapper with tr/td or we need it to not stick a table tag in there at all.